packages/mui-base/src/SliderUnstyled/SliderUnstyled.test.tsx

```diff
@@ -183,17 +210,19 @@ describe('<SliderUnstyled />', () => {
 
       const { getByRole, setProps, container } = render(<SliderUnstyled defaultValue={0} />);
 
-      stub(container.firstChild, 'getBoundingClientRect').callsFake(() => ({
-        width: 100,
-        height: 10,
-        bottom: 10,
-        left: 0,
-      }));
-
-      fireEvent.touchStart(
-        container.firstChild,
-        createTouches([{ identifier: 1, clientX: 21, clientY: 0 }]),
-      );
+      if (container.firstChild !== null) {
+        // @ts-expect-error - container.firstChild doesn't have getBoundingClientRect method. hence typescript is throwing error.
```

Comment:
No need of this line. We add ts-expect-error since TypeScript is throwing error.
```suggestion
        // @ts-expect-error - container.firstChild doesn't have getBoundingClientRect method.
```